### ISSUE TITLE
Will you release the results on CityScapes?

### ISSUE THREAD

**Issue by PkuRainBow** [CSAILVision/semantic-segmentation-pytorch#46]
Great work!

I am wondering the reproduced performance on the CityScapes. 
It would be great if you could share the related results.

**Comment by Tete-Xiao** [comment]
We're afraid that we're short-handed so we can't do that right now. 

It shouldn't be hard. You may take a try.

**Comment by PkuRainBow** [comment]
@Tete-Xiao Thanks for your quick reply. I want to try the ResNet101, but it seems that the url you provide is valid ..... 

> http://sceneparsing.csail.mit.edu/model/pretrained_resnet/resnet101-imagenet.pth

**Comment by tonysy** [comment]
@PkuRainBow Hi, could you share your training configuration?  I try to train PSP on CIityScapes, but get 68 mIOU. I use crop instead of resizing the image and the crop size is 512*512. 
Thanks a lot!

**Comment by Tete-Xiao** [comment]
@PkuRainBow  Good to hear that! We're glad to know our toolbox is useful.

**Comment by PkuRainBow** [comment]
@tonysy My crop size is 769X769

**Comment by boundles** [comment]
@PkuRainBow Hi, could you share your training configuration, such as learning rate, data augmentation?

**Comment by PkuRainBow** [comment]
@boundles  We are preparing for a new work,  which is simple and performs better than PSP.  We will release our paper and code in the future months. Please pay attention to my git repo.
